__tests__/app.test.js

```diff
@@ -71,6 +71,7 @@ describe("Reviews TESTS", () => {
               category: "dexterity",
               owner: "philippaclaire9",
               created_at: expect.any(String),
+              comment_count: "3",
```

Comment:
Better to have a new test block for this updating rather than modifying the previous test

Reply:
i thot because its refactor we are using same test, plus it's adding only 1 line